src/_premake_init.lua

```diff
@@ -1118,6 +1130,24 @@
 	end)
 
 
+	api.deprecateValue("flags", "NoExceptions", nil,
+	function(value)
+		exceptionhandling "Off"
+	end,
+	function(value)
+		exceptionhandling "On"
+	end)
```

Comment:
Here we can see that `exceptionhandling` is explicitly defined either way, even if the user never expressed an opinion.